src/UploadedFile.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
```

Comment:
Actually @jeremeamia how do you think its best to handle such a thing? there are like more than 80% changes to that file, and what's left is the obvious stuff that would have been like that either way. Do you think we need to leave the copyright message in cases like this?